.github/workflows/docker-image.yml

```diff
@@ -47,6 +48,14 @@ jobs:
         with:
           username: ${{ env.DOCKERHUB_USER }}
           password: ${{ env.DOCKERHUB_TOKEN }}
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: us-east-1
```

Comment:
this has to be `us-east-1` as this is the only region that supports public ECR repos